tpl_functions.php

```diff
@@ -910,6 +912,12 @@ function get_avatar( $username, $email, $size = 80, $d = 'mm', $r = 'g' ) {
     return $logo;
   }
 
+  if ($avatar_provider == 'activedirectory') {
+    $logo       =  "data:image/jpeg;base64," . base64_encode($INFO['userinfo']['thumbnailphoto']);
```

Comment:
I had to change this to have the uppercase "P" on thumbnailphoto, after that, it's working.
`$logo       =  "data:image/jpeg;base64," . base64_encode($INFO['userinfo']['thumbnailPhoto']);`

Also had already done the   `$conf['plugin']['authad']['additional']         = 'department,office,thumbnailPhoto';`  on local.protected.php

Reply:
Depends on what you've written in the AD config, I guess the issue comes as Active Directory is case insensitive, but PHP is case sensitive.
I'm fine with this, don't know if there should be a disclaimer somewhere that the synced property in AD config for dokuwiki should be "thumbnailPhoto" then :)